unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1747,6 +1751,14 @@ notifyUser dir o = case o of
 --      ns targets = P.oxfordCommas $
 --        map (fromString . Names.renderNameTarget) (toList targets)
 
+prettyShareLink :: WriteShareRemotePath -> Pretty
+prettyShareLink WriteShareRemotePath {repo, path} =
+  let encodedPath =
+        Path.toList path
+          & fmap (URI.encodeText . NameSegment.toText)
+          & Text.intercalate "/"
+   in P.green . P.text $ "https://share-next.unison-lang.org/users/" <> repo <> "/code/latest/namespaces/" <> encodedPath
```

Comment:
@ChrisPenner maybe pull this url out into a constant somewhere, even just in this file?